synapseAnnotations/data/curatedData.json

```diff
@@ -38,5 +38,18 @@
         "source": "http://semanticscience.org/resource/CHEMINF_000123"
       }
     ]
+  },
+  {
+    "name": "referenceSet",
+    "description": "",
```

Comment:
I would suggest this description (paraphrased from old GA4GH [schemas](https://github.com/ga4gh/ga4gh-schemas/blob/master/src/main/proto/ga4gh/references.proto)):

> "A set of references (e.g., canonical assembled contigs) which defines a common coordinate space for comparing reference-aligned experimental data."